src/nvim/os/os_defs.h

```diff
@@ -1,10 +1,90 @@
 #ifndef NVIM_OS_OS_DEFS_H
 #define NVIM_OS_OS_DEFS_H
 
+#include <ctype.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <sys/stat.h>
+#include <sys/types.h>
+
 #ifdef WIN32
 # include "nvim/os/win_defs.h"
 #else
 # include "nvim/os/unix_defs.h"
 #endif
 
+#ifdef HAVE_SYS_TIME_H
+# include <sys/time.h>
+#else
+# include <time.h>
+#endif
+
+#ifndef FILETYPE_FILE
+# define FILETYPE_FILE  "filetype.vim"
+#endif
+
+#ifndef FTPLUGIN_FILE
+# define FTPLUGIN_FILE  "ftplugin.vim"
+#endif
+
+#ifndef INDENT_FILE
+# define INDENT_FILE    "indent.vim"
+#endif
+
+#ifndef FTOFF_FILE
+# define FTOFF_FILE     "ftoff.vim"
+#endif
+
+#ifndef FTPLUGOF_FILE
+# define FTPLUGOF_FILE  "ftplugof.vim"
+#endif
+
+#ifndef INDOFF_FILE
+# define INDOFF_FILE    "indoff.vim"
+#endif
+
+#define DFLT_ERRORFILE          "errors.err"
+
+// Unix has plenty of memory, use large buffers.
+// Size of the command processing buffer.
+#define CMDBUFFSIZE 1024
+
+// Use up to 5 Mbyte for a buffer.
+#ifndef DFLT_MAXMEM
+# define DFLT_MAXMEM (5*1024)
+#endif
+// use up to 10 Mbyte for Vim.
+#ifndef DFLT_MAXMEMTOT
+# define DFLT_MAXMEMTOT (10*1024)
```

Comment:
We've been consistent about only using `//` for new/modified code, so I think it's fine as is.